server/test/routes/article.test.js

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable */
+import chai from 'chai'
+import chaiHttp from 'chai-http'
+import server from '../../index';
+import models from '../../database/models';
+import tokenService from '../../utils/services/tokenService';
+import { User } from '../../database/models';
+import mockData from './mockData';
+
+let token;
+const { expect } = chai;
+const baseUrl = '/api/articles/';
+const SIGN_UP_ROUTE = '/api/users';
+chai.use(chaiHttp);
+let slug;
+
+describe('Testing articles routes', () => {
+  
+    before((done) => {
```

Comment:
Nice work, but it will be good to use two space indentation to be consistent with the rest of the codebase